broker/broker-undelivered-messages-ui/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "message-broker",
+  "version": "1.0.0",
+  "description": "Configurations and controls for the message broker.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "pretest": "standard ./src/**/*.js",
+    "prebuild": "mkdirp ./target/webapp",
+    "build": "browserify ./src/main/webapp/index.js > ./target/webapp/bundle.js",
+    "start": "hotreloadify src/main/webapp/index.js --index=src/main/resources/index.html --proxy=https://localhost:8993 --open",
+    "format": "standard ./src/**/*.js --fix"
```

Comment:
Once you get the checkstyle running, you can run `yarn run format` from this directory to run this script to auto-fix most of the indentation and spacing issues so you don't have to manually do that. After that, `yarn test` or doing a build should point out anything else that needs to be fixed by hand.